src/components/forms/VpnServerForm.js

```diff
@@ -71,21 +51,6 @@ class VpnServerForm extends Component {
     this.setState({ [name]: event });
   }
 
-  /**
-   * Handle input change for  client_dns_server_ips text field
-   * @param {event} event
-   */
-  handleAllowedIps(event) {
-    // removing white space and checking for empty value
-    let value = event.target.value.replace(/\s*/g, "");
-    if (value === "") value = null;
-    this.setState({ client_dns_server_ips: value });
-  }
-
-  handleMultiSelectChange(name, value) {
-    this.setState(this.setNameToValue(name, value));
-  }
-
```

Comment:
i dont think you meant to remove this?
storybook can't load - getting an error due to a missing function that is bound (i think this)

Reply:
Good catch, must've deleted this by accident!